skimage/morphology/gray.py

```diff
@@ -332,7 +382,10 @@ def closing(image, footprint=None, out=None):
         Image array.
     footprint : ndarray, optional
         The neighborhood expressed as an array of 1's and 0's.
-        If None, use cross-shaped footprint (connectivity=1).
+        If None, use cross-shaped footprint (connectivity=1). This can also
+        be a sequence of 2-tuples where the first element of each 2-tuple is a
+        footprint and the second element as an integer describing the number of
+        times it should be iterated.
```

Comment:
:slightly_smiling_face: 

Also valid to the other ones below this. :joy:

Reply:
okay, I get the idea :grin:. will update these